draft-ietf-sframe-enc.md

```diff
@@ -1557,3 +1585,85 @@ The other values in the test vector are intermediate values provided to
 facilitate debugging of test failures.
 
 {::include test-vectors/sframe.md}
+
+# Acknowledgements
+{: numbered="false"}
+
+The authors wish to specially thank {{{Dr. Alex Gouaillard}}} as one of the early
+contributors to the document. His passion and energy were key to the design and
+development of SFrame.
+
+<!--[rfced] Terminology.
+
+a) FYI, we have added expansions for abbreviations upon first use per Section 3.6 of RFC 7322 ("RFC Style Guide"). Please review each expansion in the document carefully to ensure correctness.
+
+b) Please help us expand or define the following term:
+
+   SVC
+
+c) May we make the use of the acronym KID consistent?
+
+   Key ID (13) / KID (85)
```

Comment:
Generally, Key ID should be used throughout the section defining the header format, and KID elsewhere.  I have tried to implement this change.